plugins/synaptics-rmi/fu-synaptics-rmi-v7-device.c

```diff
@@ -443,9 +618,13 @@ fu_synaptics_rmi_v7_device_write_firmware(FuDevice *device,
 	fu_progress_set_id(progress, G_STRLOC);
 	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED);
```

Comment:
I push the code to change the number to what I see in the daemon output, but I don't think it'll be the same when I update different firmware or sensors.
Having different sets won't solve this problem because the time would change based on partition existence or size, or even firmware response time, should I set it to another flag like NO_PROFILE?

Reply:
> but I don't think it'll be the same when I update different firmware or sensors

Do a "typical firmware update" and use those, and then you can use NO_PROFILE to avoid the warning.

Reply:
My mistake, you weren't listed as a committer -- I've added you now. Can you push to the origin remote and I can show you want I mean?

Reply:
I pushed this branch to wip/vhuag/linux-add-verification-20230111.
Thanks.

Reply:
I've added a commit that fixes up a few nits -- can you test with V7, V8 and V9 bootloaders please, and add the right numbers where I've added FIXME. then LGTM. Thanks!

Reply:
I can't find V7 device to check the values so keep the guess flag on V7 for now, the others are updated.
Thanks